public/index.ts

```diff
@@ -6,6 +6,7 @@
 import { PluginInitializerContext } from "opensearch-dashboards/public";
 import { IndexManagementPlugin } from "./plugin";
 import { Action, UIAction } from "../models/interfaces";
+import "@opensearch-project/oui/dist/oui_theme_light.css";
```

Comment:
The theme and its css should already be imported in OSD core, I do not think we should explicitly import theme.css in a plugin.

Reply:
Yup @SuZhou-Joe. I got rid of `OUI`(faced issues with the library and documentation) entirely and switched to `EuiDataGrid`.
I am still working on the pagination part for the table. I will push the latest code with screenshots as I get some time to test for edge cases.

Thanks,
KC

Reply:
Yup @SuZhou-Joe. I got rid of `OUI`(faced issues with the library and documentation) entirely and switched to `EuiDataGrid`.
I am still working on the pagination part for the table. I will push the latest code with screenshots as I get some time to test for edge cases.

Thanks,
KC